concent_api/command_line_tool/tests/test_message_extractor.py

```diff
@@ -27,8 +28,8 @@ def setUp(self):
         self.requestor_ethereum_public_key = "b'T8ER1hrI9hH0Zyu/m6u9H2K4rc1mC/dlCqK0PSzBSwvKBdHfysmrIIsQMyvecDU+TIkhDmq93Olfo5h2FSGRjw=='"
         self.requestor_public_key = "mrdOXP6Owe3i48G29RHPVU6wewvuUNuNxFB+kPTDnmI21+p5ShttCSUAbHOVm8OIUF9hEluc1+lICJZGkFSSOA=="
         self.deadline = 1510394400
-        self.subtask_id = "12"
-        self.task_id = "2"
+        self.subtask_id = str(uuid.uuid4())
```

Comment:
please use `generate_uuid()` to enforce deterministic behavior in tests (here and below)

Reply:
:+1: done